examples/SimpleGeneratorPlugin/ImportRunner.cs

```diff
@@ -0,0 +1,150 @@
+#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Zeiss.PiWeb.Api.Core;
+using Zeiss.PiWeb.Api.Rest.Common.Authentication;
+using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+using Zeiss.PiWeb.Api.Rest.HttpClient.Builder;
+using Zeiss.PiWeb.Api.Rest.HttpClient.Data;
+using Zeiss.PiWeb.Sdk.Import.ImportPlan;
+using Zeiss.PiWeb.Sdk.Import.Modules.ImportAutomation;
+
+namespace SimpleGeneratorPlugin
+{
+    public class ImportRunner : IImportRunner
+    {
+        private readonly IActivityService _ActivityService;
+        private readonly ImportTarget _ImportTarget;
+
+        public ImportRunner(ICreateImportRunnerContext context)
+        {
+            _ActivityService = context.ActivityService;
+            _ImportTarget = context.ImportTarget;
+        }
+
+        public async Task RunAsync(CancellationToken cancellationToken)
+        {
+            if (_ImportTarget.Type != ConnectionType.Webservice)
+            {
+                _ActivityService.PostActivityEvent(EventSeverity.Error, "The import target is not supported");
+                return;
+            }
+
+            try
+            {
+                var uri = new Uri(_ImportTarget.ServiceAddress);
+                var authData = _ImportTarget.AuthData;
+                using var restClient = CreateDataServiceClient(uri, authData);
+                var random = new Random();

```

Comment:
There is also `Random.Shared` that can be used out of the box.

Reply:
From what I understand, Random.Shared goes the extra mile to ensure thread safety (hence the name). I don't think it is supposed to be used in a non-concurrent context.

Reply:
That's true, indeed. I think the main idea is to have a single global instance to allocate only once. 
I have no problem if you prefer to instantiate a new `Random` here and leave it as is.